Update TiDB upgrade FAQ (#3218)
pingcap/docs: faq/upgrade-faq.md

---

> title: FAQs Upgrade and After Upgrade

Reviewer: How about moving "FAQs" to the end? Because it sounds a little bit weird to me. 🤔
```suggestion
title: Upgrade and After Upgrade FAQs
```

> summary: Learn about some FAQs and their solutions during and after TiDB upgrade.

Reviewer: ```suggestion
summary: Learn about some FAQs and the solutions during and after upgrading TiDB.
```

aliases: ['/docs/dev/faq/upgrade-faq/','/docs/dev/faq/upgrade/']
---

> # FAQs Upgrade and After Upgrade

Reviewer: ```suggestion
# Upgrade and After Upgrade FAQs
```

This document introduces some FAQs and their solutions when or after you upgrade TiDB.

> ## FAQs Upgrade

Reviewer: ```suggestion
## Upgrade FAQs
```

This section lists some FAQs and their solutions when you upgrade TiDB.

> ### How are the rolling updates done?

Reviewer: ```suggestion
### What are the effects of rolling updates?
```

> When you apply rolling updates to the TiDB services, the running application is not affected. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3).  If the Pump/Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you update TiDB, Pump is also updated.

Reviewer: Remove an extra space.
```suggestion
When you apply rolling updates to the TiDB services, the running application is not affected. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3). If the Pump or Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you upgrade TiDB, Pump is also upgraded.
```

> ### How to upgrade Binary

Reviewer: ```suggestion
### How to upgrade TiDB using the binary?
```

> It is not recommended to deploy TiDB using Binary. The support for upgrading using Binary is not as friendly as using TiUP. It is recommended to [deploy TiDB using TiUP](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup).

Reviewer: If a file exist in this repository, just put the relative path and do not use the absolute link. Generally, use the corresponding version of link for each branch, so users can stay at that version.
```suggestion
It is not recommended to deploy TiDB using the binary. The TiDB support for upgrading using Binary is not as friendly as using TiUP. It is recommended to [deploy TiDB using TiUP](/production-deployment-using-tiup.md).
```

> ## FAQs After Upgrade

Reviewer: We use a sentence-style capitalization style for the body. Please refer to other documents.
```suggestion
## After upgrade FAQs
```
